applications/challengedac.md

```diff
@@ -0,0 +1,109 @@
+
+### Ecosystem Fit
+
+ChallengeDapp is a decentralized application (dApp) that runs on the EOS blockchain and it is designed to allow users to create and participate in a variety of challenges. ChallengeDapp utilizes the EOS blockchain to provide a decentralized and trustless platform where users can create and participate in Challenges. The use of smart contracts on the EOS blockchain ensures that the rules of each challenge are transparent and immutable and that the rewards for winning a challenge are automatically distributed to the winners. ChallengeDapp also utilizes the EOS token on the platform. Users can use EOS tokens to create new challenges, participate in existing challenges, and receive rewards for winning challenges. This dApp fits into the EOS ecosystem by providing a platform that can be used to create and participate in various challenges, and by incentivizing users to engage with the EOS blockchain and use EOS tokens. Additionally, it offers entertainment and educational opportunities for users and it can be seen as a way to increase the adoption and usage of the EOS blockchain.
+
+## Team
+
+### Team members
+
+- **Team Leader:** Chace Eskam
+- **Co-founder:** Kent Eskam
+- **System Engineer:** Phung Ta
+
+### Legal Structure
+
+- **Registered Legal Entity:** CALI-PUMP LLC
+- **Registered Address:** 300 Carlsbad Village Dr, Carlsbad, CA 92008
+
+### Team Experience
+
+
+Chace Eskam: Has spent a number of years studying the social and economic impact of the blockchain. Chace has given talks about blockchains future implications on business at W.P Carey school of business as early as February 2016. Before devoting himself fully to blockchain Chace spent a number of years working in the fintech industry at the Nations first digital bank.
+
+### Team Org Repos
+
+- https://github.com/ChallengeDAC
+- https://github.com/ChallengeDAC/dacdart
+
+## Development Roadmap
+
+### Milestone Summary
+
+- **Total Estimated Duration:** 6 months 
+- **Full-Time Equivalent (FTE):** 1 FTE
+- **Total Costs:** 18,500 USD
+
+### Milestone 1 - Develop DacDart, a decentralized SDK, using EOS Node API to hook to ChallengeDac application blocks to view accounts, performs smart contracts
```

Comment:
By the name of it, it appears that this will be residing on the client (iOS/Android device). Can you please elaborate why any changes to nodeos rpc api and/or eosjs deemed to be necessary and if so, what are the plans of running modified  backend services?

Reply:
Do you have any insight on that?

Reply:
Please refer to this [answer](https://github.com/eosnetworkfoundation/grant-framework/pull/150#discussion_r1211194076), thanks!

Reply:
Ok, thank you, marking this as resolved to avoid duplicates, but question about need for refactoring existing node api still stays in the question here  https://github.com/eosnetworkfoundation/grant-framework/pull/150#discussion_r1187848200